API-Reference/Kokkos::Subview.md

```diff
@@ -0,0 +1,42 @@
+# `Kokkos::Subview`
+
+Header File: `Kokkos_Core.hpp`
+
+Alias template to deduce the type that is returned by a call to the subview function with given arguments.
+
+Usage:
+  ```c++
+  Kokkos::Subview<ViewType,Args> subView;
+  ```
+
+## Description
+
+  ```c++
+  template <class ViewType, class... Args>
+  using Subview = IMPL_DETAIL // deduce subview type from source view traits
+  ```
+  Type of a `Kokkos::View` viewing a subset of `ViewType` specified by `Args...`.
+  Same type as returned by a call to the subview function with corresponding arguments.
+  For restrictions on Args see subview documentation.
```

Comment:
Maybe we should link to `Kokkos::subview` page here

Reply:
I did not see any links in the other subpages of the API-View ... I don't know if this is a requirement or was a deliberate decision but that was the reason I did not. (If we want to add them, then we should even make the mentions of subview or other functions in the API of View etc. links ... this would be a bigger change though)

Reply:
Yeah make a link here, and change the "see subview" to "see `Kokkos::subview`"

Reply:
Done in [ad0118](https://github.com/kokkos/kokkos-core-wiki/pull/75/commits/ad01187c6c6395380064f1df61f0d4aa8b00807c)